### PR TITLE
Improve README and documents with AI. v5.0.153. v6.0.43

### PR DESCRIPTION
**Pull request by winlinvip**
1. Update README file with AI to make it more informative and user-friendly
2. Add a detailed table of contents (TOC) with an introduction for easy navigation
3. Introduce an auto-detecting Automake feature that displays the correct installation command
4. Add support for SRT to HTTP-TS config file
5. Refine the WHIP delete location URL
6. Add support for disabling encryption for WHIP or WHEP

This pull request aims to enhance the quality of the project by introducing innovative features and making the necessary updates. These updates will help users navigate the project more efficiently while also improving the overall project's quality.

### CODE REVIEW

**Review by xiaozhihong — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,104 +18,113 @@
...
 
 ```
 git clone -b develop https://gitee.com/ossrs/srs.git &&
 cd srs/trunk && ./configure && make && ./objs/srs -c conf/srs.conf
 ```
 
-Open [http://localhost:8080/](http://localhost:8080/) to check it, then publish
-by [FFmpeg](https://ffmpeg.org/download.html) or [OBS](https://obsproject.com/download) as:
+Open [http://localhost:8080/](http://localhost:8080/) to verify, and then stream using follwing
+[FFmpeg](https://ffmpeg.org/download.html) command:
```
Comment (chundonglinlin) [inline README.md]: There is an extra space here

`TRANS_BY_GPT3`

File: README.md [hunk 2]
```diff
@@ -17,104 +18,113 @@
...
+To compile SRS from source:
 
 ```
 git clone -b develop https://gitee.com/ossrs/srs.git &&
 cd srs/trunk && ./configure && make && ./objs/srs -c conf/srs.conf
 ```
 
-Open [http://localhost:8080/](http://localhost:8080/) to check it, then publish
-by [FFmpeg](https://ffmpeg.org/download.html) or [OBS](https://obsproject.com/download) as:
+Open [http://localhost:8080/](http://localhost:8080/) to verify, and then stream using follwing
```
Comment (chundonglinlin) [inline README.md]: `using follwing` should be changed to `using the following`

File: README.md [hunk 3]
```diff
@@ -17,104 +18,113 @@
...
+> To enable WebRTC to publish and convert it to RTMP, please refer to the wiki([CN](https://ossrs.net/lts/zh-cn/docs/v4/doc/webrtc#rtc-to-rtmp), [EN](https://ossrs.io/lts/en-us/docs/v4/doc/webrtc#rtc-to-rtmp)) documentation.
+> It is essential to ensure the candidate([CN](https://ossrs.net/lts/zh-cn/docs/v4/doc/webrtc#config-candidate) or [EN](https://ossrs.io/lts/en-us/docs/v4/doc/webrtc#config-candidate)) 
+> is set correctly for WebRTC to avoid potential issues, as it can cause significant problems.
+
+> Note: It is highly recommended to run SRS directly with docker([CN](https://ossrs.net/lts/zh-cn/docs/v4/doc/getting-started) / [EN](https://ossrs.io/lts/en-us/docs/v4/doc/getting-started)), 
+> CVM([CN](https://ossrs.net/lts/zh-cn/docs/v4/doc/getting-started-cloud) / [EN](https://ossrs.io/lts/en-us/docs/v4/doc/getting-started-cloud)), 
+> or K8s([CN](https://ossrs.net/lts/zh-cn/docs/v4/doc/getting-started-k8s) / [EN](https://ossrs.io/lts/en-us/docs/v4/doc/getting-started-k8s)). 
+> However, building SRS from source code is also possible and easy. For detailed instructions, please refer to the
```
Comment (chen-guanghua) [inline README.md]: 38 lines have been modified to: To compile SRS from source. Should the word "build" here be updated to "compiling"?

`TRANS_BY_GPT3`

Comment (winlinvip) [reply README.md]: Both sentences are acceptable and convey the same meaning, but "compiling" would be a more common word choice in this context. So, "Compiling SRS from source code is also possible and easy." would be more appropriate and clear.

**Review by chen-guanghua — APPROVED** [review]

**Pull request merged**